guides/v2.1/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -81,8 +81,12 @@ Verify that the modified source displays correctly in the browser. Then, add the
 
 1.	Click **Save Config** at the top of the page to save the updated configuration file.
 
+1. Refresh the cache.
+
+   - In the notification at the top of the page, click the **Cache Management** link.
+   
+   - On the Cache Management page, click **Flush Magento Cache**.
```

Comment:
Go ahead an italicize "Cache Management" because it is identifying a location you want them to notice, like a page title or a tab name. All locations can be italicized for emphasis.